framework/src/modules/chain/chain.js

```diff
@@ -107,8 +107,8 @@ module.exports = class Chain {
 				lastCommit,
 				ed,
 				build: versionBuild,
-				config: this.options.config,
-				genesisBlock: { block: this.options.config.genesisBlock },
+				config: self.options.config,
```

Comment:
I mean remove `self` and use this instead 😕

Reply:
As commented earlier, we have to use self to pass reference to sequence. Otherwise have to bind this to sequence events.